editor/src/components/editor/actions/actions.ts

```diff
@@ -4531,6 +4544,61 @@ export const UPDATE_FNS = {
       },
     }
   },
+  ADD_TAILWIND_CONFIG: (
+    action: AddTailwindConfig,
+    editor: EditorModel,
+    dispatch: EditorDispatch,
+  ): EditorModel => {
+    const packageJsonFile = getContentsTreeFileFromString(editor.projectContents, '/package.json')
+    const currentNpmDeps = dependenciesFromPackageJson(packageJsonFile, 'regular-only')
+    const updatedNpmDeps = [
+      ...currentNpmDeps,
+      requestedNpmDependency('postcss', PostCSSVersion),
+      requestedNpmDependency('tailwindcss', TailwindCSSVersion),
```

Comment:
I think we can add these without specifying a version (so that it takes the latest version available)

Reply:
Also, thinking about it, we should probably check that these aren't already in the list of dependencies

Reply:
I changed this to load the latest version